src/metpy/testing.py

```diff
@@ -23,37 +24,89 @@
 from .deprecation import MetpyDeprecationWarning
 from .units import units
 
-MPL_VERSION = Version(matplotlib.__version__)
 
+def module_version_check(version_spec):
```

Comment:
I think this got lost in the comments above, but I don't think there's really any ambiguity in looking at code that says:
```python
version_check('pint < 0.21')
```
so let's go with the shortened `version_check`.